mock/mock.spec

```diff
@@ -3,6 +3,9 @@
 
 %global __python %{__python3}
 %global python_sitelib %{python3_sitelib}
+%if 0%{?rhel} == 7
```

Comment:
and if python3_pkgversion equals to 3?
So if EPEL bumps up python3_pkgversion we do not need to change it here.

Reply:
You can not build it on RHEL 7 host, with EPEL 7 repo enabled, unless you also
install epel-rpm-macros ... and this problem is pretty sneaky to realize what is
going on ....